linux-anvil-cos7/Dockerfile

```diff
@@ -1,24 +1,36 @@
+# Set `ARG`s for use in `FROM`.
+ARG DISTRO_ARCH
 ARG DISTRO_NAME
 ARG DISTRO_VER
-FROM ${DISTRO_NAME}:${DISTRO_VER}
+FROM --platform=linux/${DISTRO_ARCH} ${DISTRO_NAME}:${DISTRO_VER}
 
 LABEL maintainer="conda-forge <conda-forge@googlegroups.com>"
 
+# Set `ARG`s during build.
+ARG DISTRO_ARCH
 ARG DISTRO_NAME
 ARG DISTRO_VER
-ENV DISTRO_NAME=${DISTRO_NAME} \
+ENV DISTRO_ARCH=${DISTRO_ARCH} \
+    DISTRO_NAME=${DISTRO_NAME} \
     DISTRO_VER=${DISTRO_VER}
 
 # Set an encoding to make things work smoothly.
 ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+# bust the docker cache so that we always rerun the installs below
+ADD https://loripsum.net/api /opt/docker/etc/gibberish
```

Comment:
Any reason you moved the cache busting before the qemu binaries?

Reply:
The original idea of busting the cache was to make sure the images were rebuilt from scratch from the beginning (particularly on services like DockerHub, which always use a cache to build)

So had moved this before other steps

Though if there are other considerations, would be interested to learn more about them